multicluster/cmd/multicluster-controller/controller.go

```diff
@@ -37,8 +38,8 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/webhook"
 	k8smcsv1alpha1 "sigs.k8s.io/mcs-api/pkg/apis/v1alpha1"
 
-	multiclusterv1alpha1 "antrea.io/antrea/multicluster/apis/multicluster/v1alpha1"
-	multiclusterv1alpha2 "antrea.io/antrea/multicluster/apis/multicluster/v1alpha2"
+	mcsv1alpha1 "antrea.io/antrea/multicluster/apis/multicluster/v1alpha1"
+	mcsv1alpha2 "antrea.io/antrea/multicluster/apis/multicluster/v1alpha2"
```

Comment:
I may have missed something, but I did not see you add v1alpha2 but just saw you modified types of v1alpha1? Could you point to me how v1alpha2 is added?

Reply:
Do you mean the `v1alpha2` API? It was added by this PR https://github.com/antrea-io/antrea/pull/3755 which is done by Bangqi.
Before the change, there is a Name field in ClusterClaim which is already part of ObjectMeta and its confusing to have two names. So we removed the  Name field  in v1alpha2.